x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/NoopStep.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+package org.elasticsearch.xpack.core.ilm;
+
+/**
+ * Signals that an error was encountered during the execution of a policy on an index.
```

Comment:
I think this javadoc was copied from elsewhere and needs to be updated

Reply:
ya i just updated it to reflect the noopstep purpose correctly